src/zeusops_bot/reforger_config_gen.py

```diff
@@ -163,3 +163,11 @@ def extract_mods(modlist: str | None) -> list[ModDetail] | None:
             case _:
                 # Unknown error
                 raise e
+
+    if not strip_versions:
+        return mods
+
+    for mod in mods:
+        del mod["version"]
+
+    return mods
```

Comment:
Purely instinct, I recognize the Early Exit (and usually love it), but here for some reason I feel like that: 
```suggestion
    if strip_versions:
        for mod in mods:
            del mod["version"]

    return mods
```

Minor, take it or leave it, don't know

Reply:
Added